src/SmartComponents/Systems/SystemsListAssets.js

```diff
@@ -95,14 +95,6 @@ export const advisorySystemsColumns = () => [
 ];
 
 export const packageSystemsColumns = [
-    {
-        key: 'display_name',
```

Comment:
I think we still have to have `'display_name'` as a string in the column definition. That way the Inventory knows where we want to show the display_name column, but we won't override the default column.

... I believe `{ key: 'display_name' }` might work too.

Reply:
I believe we are safe to remove this column definition. Because, the inventory module [merges](https://github.com/RedHatInsights/insights-inventory-frontend/blob/97749de23e2d1851e3bb9dd6533af29fbe53e0c4/src/components/InventoryTable/hooks/useColumns.js#L41) those columns by default to those patch related columns that we pass in.

Reply:
I have removed tags column definition as well as it is also not necessary